packages/core/src/test/shared/sam/deploy.test.ts

```diff
@@ -142,6 +143,7 @@ describe('DeployWizard', async function () {
             assert.strictEqual(parameters.region, 'us-west-2')
             assert.strictEqual(parameters.stackName, 'stack1')
             assert.strictEqual(parameters.bucketSource, 0)
+            prompterTester.assertCallAll(prompterTester.getHandlers(), 1)
```

Comment:
seems `prompterTester.assertCallAll` is always used with `prompterTester.getHandlers()` and 1. Try making these as default so you can use `prompterTester.assertCallAll()` directly

Reply:
>  Try making these as default so you can use prompterTester.assertCallAll() directly

That what I did initially, but as pointed out by Justin, such function is not that useful as it is too specific to only one case. Instead, we should provide reusable method that user can reuse to fit different scenario. Please refer to resolve discussion above.

Reply:
I'll update the method to `prompterTester.assertCallAll(prompterTester.getHandlers(), 1)` is default case if no parameters provided.